x-pack/plugins/security_solution/public/flyout/document_details/left/tabs/visualize_tab.tsx

```diff
@@ -86,6 +122,22 @@ export const VisualizeTab = memo(() => {
     }
   }, [panels.left?.path?.subTab]);
 
+  // Decide whether to show the graph preview or not
+  const { isAuditLog } = useGraphPreview({
```

Comment:
this `isAuditLog` is a bit confusing, because it's not clear what audit log has to do with graph. Should we encapsulate it into less specific flag, smth like `hasGraphRepresentation` or smth? In general I think `audit` is quite loaded term, because we have audit logs for actions in kibana itself, we also have different audit integrations, like `auditd` and it's not clear why the following
```
const isAuditLog =
    Boolean(timestamp) &&
    Boolean(action?.length) &&
    actorIds.length > 0 &&
    eventIds.length > 0 &&
    targetIds.length > 0;
```
is the definition of an audit log